src/term/tui.rs

```diff
@@ -95,13 +104,19 @@ impl<B: Backend> Tui<B> {
 	///
 	/// It disables the raw mode and reverts back the terminal properties.
 	pub fn exit(&mut self) -> Result<()> {
+		Self::reset()?;
+		self.terminal.show_cursor()?;
+		Ok(())
+	}
+
+	fn reset() -> Result<()> {
 		terminal::disable_raw_mode()?;
 		crossterm::execute!(
-			io::stderr(),
+			io::stdout(),
```

Comment:
I think this should be `stderr` instead since the terminal runs on `stderr`:

https://github.com/orhun/gpg-tui/blob/7975a47ab995cf5fddbcd49cf23ea399e46e085f/src/main.rs#L37